packages/design-system/.storybook/main.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
 		buildStoriesJson: true,
 		modernInlineRender: true,
 		previewCsfV3: true,
+		// storyStoreV7: true, // will break all work related to aggregated status in the next major version of Storybook
```

Comment:
Do we need to keep the commented code + comment? Seems weird to have a warning on top of unused code.

Reply:
I would like to keep that because I need to think about it for the next upgrades if it's ok for you